tests/core/v5_1/alexandria/test_content_storage.py

```diff
@@ -55,6 +76,351 @@ def test_content_storage_read_write_exists_delete(content_storage):
         content_storage.delete_content(content_key)
 
 
+def test_content_storage_set_content_exists_ok(content_storage):
+    content_key = b"\x00test-key"
+    content = ContentFactory(128)
+
+    assert content_storage.has_content(content_key) is False
+
+    content_storage.set_content(content_key, content)
+
+    with pytest.raises(ContentAlreadyExists):
+        content_storage.set_content(content_key, content)
+    with pytest.raises(ContentAlreadyExists):
+        content_storage.set_content(content_key, content, exists_ok=False)
```

Comment:
Is this test redundant / testing the exact same thing as the line above (since `exists_ok` defaults to `False`)? Maybe it's still useful to explicitly test this path...